File: font-end/src/components/services/sizeService.js
import axios from "axios";


class SizeService{
    getAll=()=>{
        return axios.get(`http://localhost:8080/size/list`)
    }

    addSize=(sizeRequest)=>{
        return axios.post(`http://localhost:8080/size/add`,sizeRequest)
    }
}

export default new SizeService();

File: back-end/src/main/java/com/nvm/project1/service/SizeService.java
package com.nvm.project1.service;

import com.nvm.project1.entity.Size;
import com.nvm.project1.request.SizeRequest;
import com.nvm.project1.response.SizeResponse;

import java.util.List;

public interface SizeService {

    List<SizeResponse> getAll();

    SizeResponse add(SizeRequest sizeRequest);


}
